via/via_system.cpp

```diff
@@ -253,7 +253,12 @@ ViaSystem::~ViaSystem() {
 #ifdef VIA_WINDOWS_TARGET
     if (_out_file_format == VIA_HTML_FORMAT) {
         // Open the html file in a browser
+        // Sleep 3 seconds before bringing up the browser so console window is visible for a little while
+        Sleep(3000);
         ShellExecute(NULL, "open", _full_out_file.c_str(), NULL, NULL, SW_SHOWNORMAL);
+        // Sleep another 7 seconds after bringing up the browser. The console probably won't be visible,
+        // it will be covered by browser. But it's available on the screen for a while for viewing if needed.
+        Sleep(7000);
```

Comment:
I'm conflicted about the delays, as while they do make the console a bit more visible, expecting people to be able to read it in the allotted time is expecting too much. 
If we want to make sure its visible, why close the console at all? Or put another way, is the delayed closing meant to allow people to *read* whats going on, or simply see that its happening?

Reply:
Made changes to address this. Removed the sleep, and now via asks user to press enter before closing the console window.